pyanaconda/bootloader/grub2.py

```diff
@@ -183,10 +183,12 @@ def write_config_images(self, config):
 
     @property
     def stage2_format_types(self):
-        if productName.startswith("Red Hat "): # pylint: disable=no-member
-            return ["xfs", "ext4", "ext3", "ext2"]
-        else:
-            return ["ext4", "ext3", "ext2", "btrfs", "xfs"]
+        # If the product file_system_type is valid, use it by default
+        valid_fs = ["ext4", "ext3", "ext2", "btrfs", "xfs"]
```

Comment:
`btrfs` is not a valid format type for RHEL.

Reply:
Perhaps the best way to do that is add an 'invalid_fs' to the Products?

Grub2 seems to work with `btrfs`, so I wouldn't want to drop it from the supported grub2 fs list.

Reply:
It looks like it will not be necessary to change `stage2_format_types` at all, so I would prefer to leave it as it is for now.